setup.cfg

```diff
@@ -0,0 +1,32 @@
+
+[metadata]
+name = jnc
+summary = Pyang plugin generating classes for Java Netconf Client
```

Comment:
maybe we should stabilize JNC name with capital NETCONF lettering across repo, not to introduce "incorrect" Netconf form?

Reply:
fixed